src/CallHandler.tsx

```diff
@@ -475,9 +478,22 @@ export default class CallHandler extends EventEmitter {
                 this.silencedCalls.delete(call.callId);
             }
 
+            const incomingCallPushRule = (
```

Comment:
Seems like this can be moved directly to the `CallState.Ringing` case, as it's not used anywhere else. Should be properly scoped due to `const` anyways.

Reply:
I can't do that, we seem to have https://eslint.org/docs/rules/no-case-declarations enabled

Reply:
@SimonBrandner answer's here, you just need to make a block scope. :D https://stackoverflow.com/a/50753272